compiler/rustc_middle/src/query/mod.rs

```diff
@@ -115,6 +118,12 @@ rustc_queries! {
         desc { "triggering a delayed bug for testing incremental" }
     }
 
+    query expand_legacy_bang(key: (LocalExpnId, Span, LocalExpnId)) -> Result<(&'tcx TokenStream, usize), CanRetry> {
+        eval_always
+        no_hash
```

Comment:
Do you still need all extra `Hash` impls with the `no_hash` modifier enabled?

Reply:
That's right, thanks! I have cleaned them